modules/FvwmRearrange/FvwmRearrange.c

```diff
@@ -402,8 +402,15 @@ void parse_args(char *s, int argc, char *argv[], int argi)
   /* parse args */
   for (; argi < argc; ++argi)
   {
-    if (!strcmp(argv[argi], "-tile") || !strcmp(argv[argi], "-cascade")) {
-      /* ignore */
+    if (!strcmp(argv[argi], "-tile")) {
+      FvwmTile = 1;
+      FvwmCascade = 0;
+      resize = 1;
+    }
+	/* Does this do anything since this is default action? */
+    else if (!strcmp(argv[argi], "-cascade")) {
+      FvwmCascade = 1;
+      FvwmTile = 0;
```

Comment:
Hey @somiaj 

Hmm.  I think this should be the default (set before the check for `"-tile"`, above) -- and even if someone does pass `-cascade` through, it's a no-op as these values are already set.

Reply:
Yea, it should be the default, was more wondering if this is needed, though right now if someone did `FvwmRearrange -title -cascade` it would then set it back.